practice/00-HTML-CSS-basics/topic0.1v.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Image, video and sound</title>
+  </head>
+  <body>
+    <h1>Image</h1>
+
+
+<img src="http://cdn01.ib.infobae.com/adjuntos/162/imagenes/012/198/0012198196.jpg"/>
+
+<h1>Video</h1>
+
+<embed src="video topic 01v.mp4" width="300" height="300">
```

Comment:
For audio and video, use the html5 tags audio and video.

https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML5_audio_and_video